### PR TITLE
Update summarize-youtube-video-with-ai extension

### PR DESCRIPTION
**Pull request by iKasch**
## Description

This PR adds new functionality to the summarize youtube extension:

- 🤖 Added Anthropic Claude
- 🔎 Added Action to ask follow up questions
- 🧑‍💻 Improved AI instruction snippets
- 🔧 Added up to date AI models as defaults
  - Raycast AI: Your selected AI Model
  - OpenAI: `gpt-4o`
  - Anthropic: `claude-3-5-sonnet-latest`
- 🐛 Fixed issues with long loading time
- 🖼️ Added new Icon

It will fix: #15922 #14855 

## Screencast

https://github.com/user-attachments/assets/6c09e95b-8c34-483e-bb90-73a37f8a7243

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/summarize-youtube-video-with-ai/.eslintrc.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
   "root": true,
-  "extends": ["@raycast"]
-}
\ No newline at end of file
+  "extends": ["@raycast"],
```
Comment (pernielsentikaer) [inline extensions/summarize-youtube-video-with-ai/.eslintrc.json:3]: Can we keep as it was?

Comment (iKasch) [reply extensions/summarize-youtube-video-with-ai/.eslintrc.json:3]: @pernielsentikaer yea, I saw the check failing. I'm used to not have semicolons and some other QoL checks but I'll adjust.

Comment (pernielsentikaer) [reply extensions/summarize-youtube-video-with-ai/.eslintrc.json:3]: It's just so we keep the same with all extensions 🙂

Comment (iKasch) [reply extensions/summarize-youtube-video-with-ai/.eslintrc.json]: Got it! I reset the config and checks are green now 😊

**Comment by pernielsentikaer** [comment]
Thanks 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**